org.adempiere.base/src/org/compiere/acct/Doc_AllocationHdr.java

```diff
@@ -1091,12 +1105,12 @@ private String createInvoiceRoundingCorrection (MAcctSchema as, Fact fact, MAcco
 				.append(" FROM Fact_Acct ")
 				.append("WHERE AD_Table_ID=? AND Record_ID=?")
 				.append(" AND C_AcctSchema_ID=?")
-				.append(" AND Account_ID=?")
+				.append(" AND Account_ID in ( "+commaseparatedlist+" ) ")
```

Comment:
old logic, each invoice need one account to query fact, so i like to keep that logic